cfme/tests/test_db_migrate.py

```diff
@@ -21,13 +20,40 @@ def pytest_generate_tests(metafunc):
     return metafunc.parametrize(argnames=argnames, argvalues=argvalues, ids=idlist)
 
 
+@pytest.fixture(scope="session")
+def extend_db_partition():
+    app = pytest.store.current_appliance
```

Comment:
please use the get_or_create function that backs the current_appliance context local in that place